src/utils/__tests__/indexForSearch.test.ts

```diff
@@ -45,6 +46,12 @@ describe("indexForSearch", () => {
         },
       })
     )
+    originalConsoleLog = console.log
+    console.log = jest.fn()
+  })
+
+  afterEach(() => {
+    console.log = originalConsoleLog
```

Comment:
The one opinionated change I made was to keep the test runs free of console output. If this output was in fact useful, I'm happy to revert this.

---
#### Before
---

![before](https://user-images.githubusercontent.com/140521/59875045-c48ffd80-936d-11e9-9c16-861afccc89f3.png)

---
#### After
---

![after](https://user-images.githubusercontent.com/140521/59875056-c954b180-936d-11e9-8136-4742c0404178.png)